src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceFacadeImpl.java

```diff
@@ -125,28 +127,31 @@
                 }
             }
 
-            List<ExceptionCaseWorker> exceptionCaseWorkerList =
-                auditAndExceptionRepositoryServiceImpl.getAllExceptions(jobId);
-
-            status = (nonNull(exceptionCaseWorkerList) && (exceptionCaseWorkerList.size()) > 0)
-                ? PARTIAL_SUCCESS : status;
-
-            validationServiceFacadeImpl.insertAudit(status, fileName);
-
-            CaseWorkerFileCreationResponse caseWorkerFileCreationResponse =
-                createResponse(totalRecords, exceptionCaseWorkerList, isCaseWorker);
-
-            return ResponseEntity.ok().body(caseWorkerFileCreationResponse);
+            return sendResponse(file, status, isCaseWorker, totalRecords);
 
         } catch (Exception ex) {
-            long jobId = validationServiceFacadeImpl.insertAudit(AuditStatus.FAILURE, file.getOriginalFilename());
-            ExceptionCaseWorker exceptionCaseWorker = validationServiceFacadeImpl.createException(jobId,
-                ex.getMessage(), 0L);
-            auditAndExceptionRepositoryServiceImpl.auditException(exceptionCaseWorker);
+
+            long jobId = validationServiceFacadeImpl.updateCaseWorkerAuditStatus(AuditStatus.FAILURE,
+                file.getOriginalFilename());
+            validationServiceFacadeImpl.logFailures(ex.getMessage(), 0L);
+            log.error("{}:: Failed File Upload for job {}", loggingComponentName, jobId);
             throw ex;
         }
     }
 
+    public ResponseEntity<Object> sendResponse(MultipartFile file, AuditStatus status,
+                                                Boolean isCaseWorker, int totalRecords) {
+        List<ExceptionCaseWorker> exceptionCaseWorkerList = exceptionCaseWorkerRepository
+            .findByJobId(validationServiceFacadeImpl.getAuditJobId());
+        CaseWorkerFileCreationResponse caseWorkerFileCreationResponse =
+            createResponse(totalRecords, exceptionCaseWorkerList, isCaseWorker);
+        status = (nonNull(exceptionCaseWorkerList) && (exceptionCaseWorkerList.size()) > 0)
+            ? PARTIAL_SUCCESS : status;
+        long jobId = validationServiceFacadeImpl.updateCaseWorkerAuditStatus(status, file.getOriginalFilename());
+        log.info("{}:: Completed File Upload for Job {} with status", loggingComponentName, jobId, status);
```

Comment:
I think we need to add {} after status like 
`log.info("{}:: Completed File Upload for Job {} with status {}", loggingComponentName, jobId, status);`